src/Modules/Build.lua

```diff
@@ -156,11 +156,53 @@ function buildMode:Init(dbFileName, buildName, buildXML, convertBuild)
 		end
 	end
 	self.controls.pointDisplay.width = function(control)
-		local used, ascUsed = self.spec:CountAllocNodes()
-		local usedMax = 99 + 22 + (self.calcsTab.mainOutput.ExtraPoints or 0)
-		local ascMax = 8
-		control.str = string.format("%s%3d / %3d   %s%d / %d", used > usedMax and "^1" or "^7", used, usedMax, ascUsed > ascMax and "^1" or "^7", ascUsed, ascMax)
-		control.req = "Required level: "..m_max(1, (100 + used - usedMax))
+		local PointsUsed, AscUsed = self.spec:CountAllocNodes()
+		local bandit = self.calcsTab.mainOutput.ExtraPoints or 0 
+		local usedMax, ascMax, levelreq, currentAct, banditStr, labSuggest = 99 + 22 + bandit, 8, 1, 1, "", ""
```

Comment:
I'm fine with this, but be careful of stringing too many declarations together at once, especially if they're unrelated, as it's harder to parse vs. just having a couple related ones per line.